appserver/web/gf-web-connector/src/main/java/fish/payara/appserver/context/JavaEEContextUtilImpl.java

```diff
@@ -69,6 +69,14 @@
 @Service
 @PerLookup
 public class JavaEEContextUtilImpl implements JavaEEContextUtil, Serializable {
+    private transient @Getter(AccessLevel.PACKAGE) ServerContext serverContext;
+    private transient ComponentEnvManager compEnvMgr;
+    private transient ComponentInvocation capturedInvocation;
+    private @Getter(onMethod = @__(@Override)) String instanceComponentId;
```

Comment:
Unless I missed something we're still not allowing lombok.

Reply:
This has been there for a long time now.  I just changed the location as per coding standards.
I will get to making a massive PR in the future to remove lombok from everything so it can be correctly evaluated.
For now, please don't hold up a regression fix for this, I am just keeping the status quo for now, not adding new lombok code